### PR TITLE
Getting started with Terraform and AKS

### PR DESCRIPTION
**Pull request by danielepolencic**
Preview: https://deploy-preview-188--learnk8s.netlify.com/blog/get-start-terraform-aks/

### CODE REVIEW

File: src/terraformAks/content.md [hunk 1]
```diff
@@ -0,0 +1,868 @@
...
+
+### Terraform
+
+[Terraform](https://www.terraform.io/intro/index.html#what-is-terraform-) gained most of its popularity from being a friendly tool to provision infrastructure on Amazon Web Services.
+
+Terraform is not a library that you use in your favourite programming language, and it's not even a collection of JSON templates.
+
+It's something in between.
+
+It's a sort of DSL — a domain specific language that's designed to be easy to read and write.
```
[the remark this answers is not in the document]
Comment (danielepolencic) [reply src/terraformAks/content.md]: Add or replace?

File: src/terraformAks/content.md [hunk 2]
```diff
@@ -0,0 +1,868 @@
...
+
+```hcl
+terraform apply
+```
+
+_It's time for a cup of coffee._
+
+Provisioning a cluster on AKS takes in average about ten minutes.
+
+What happens in the background is that Azure receives your request, generates a few ARM templates, deploys them internally and create the extra resources needed (such as NICs and virtual machines) to create the cluster.
```
[the remark this answers is not in the document]
Comment (danielepolencic) [reply src/terraformAks/content.md]: I think I got confused by how ACS-engine and AKS-engine work. I removed the part about the ARM te plates.

File: src/terraformAks/content.md [hunk 3]
```diff
@@ -0,0 +1,868 @@
...
+You have provision a cluster using Terraform.
+
+_The cluster is empty, though._
+
+_And it doesn't expose any port to the public._
+
+Let's fix that by adding a security group:
+
+```hcl|title=main.tf|highlight=36
+resource azurerm_network_security_group "sg" {
```
[the remark this answers is not in the document]
Comment (danielepolencic) [reply src/terraformAks/content.md]: I didn't even know I could do this :) Fixed

File: src/terraformAks/content.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (antonbabenko) [reply src/terraformAks/content.md]: Rephrase to include that statement :)

**Pull request merged**